any_parser/any_parser.py

```diff
@@ -221,6 +222,8 @@ def async_extract(
             process_type = ProcessType.FILE
         elif model == ModelType.PRO:
             process_type = ProcessType.FILE_REFINED_QUICK
+        elif model == ModelType.ADVANCED:
```

Comment:
nit: let's call this parse_with_layout in the ModelType.

Reply:
ack